content/docs/contributing/to-code/git-workflow.md

```diff
@@ -91,7 +91,16 @@ At a high level, the workflow is:
 
 Now, you're ready to do some work! After you've made some changes and saved your work, it's time to submit it as a pull request (PR) for review.
 
-1. First, stage (add) the new or modified code files, commit your changes, and submit a pull request to your fork (origin) with the following commands. (**Check which branch you are on with `git branch` first!**):
+1. (Optional): First, depending on how much time has passed since your branch was initially created, you might want to merge the latest from `upstream/development` into your branch. This will ensure your changes don't conflict with any other recent code commits, and also gives your automated review (AR) the best chance of success the closer your branch is to the latest development snapshot.
```

Comment:
nitpick - I think this reads a little strangely. I would suggest either "... the best chance of success, because your branch will be closer to the latest development snapshot." or "... a better chance of success the closer your branch is..."

Reply:
Good call, I like the former, I'll update it!